extras/pom.xml

```diff
@@ -34,7 +34,7 @@ under the License.
         <module>rya.prospector</module>
         <module>rya.manual</module>
         <module>rya.periodic.service</module>
-        <module>rya.console</module>
+        <module>shell</module>
```

Comment:
It makes filesystem navigation by shell faster and yields shorter, less redundant paths.  If that convention was fully expanded, we would end up having:
`incubator-rya/rya.extras/rya.shell` which is more rya's than I'd like to type.

Does windows still have those long path limitations?